tests/edge-cases/logand-07.txt

```diff
@@ -0,0 +1,2 @@
+brun '(logand (q 128) (q 128))'
```

Comment:
For the logical operators, I feel like it would be good to include a value that spans one extra byte, like 0x01ff